connectors/sources/jira.py

```diff
@@ -974,9 +990,9 @@ async def get_docs(self, filtering=None):
             )
 
             for rule in advanced_rules:
-                await self.fetchers.put(
-                    partial(self._get_issues, rule.get("query", ""))
-                )
+                query = rule.get("query", "")
+                self._logger.debug(f"Fetching issues using query: {query}")
```

Comment:
Feel it's more appropriate to put this log inside method `_get_issues`

Reply:
Sorry, missed your comment, will adapt it 👍